pkgs/top-level/all-packages.nix

```diff
@@ -10835,6 +10835,10 @@ with pkgs;
 
   nginxModules = callPackage ../servers/http/nginx/modules.nix { };
 
+  nginxShibboleth = callPackage ../servers/http/nginx/stable.nix {
+    modules = [ nginxModules.rtmp nginxModules.dav nginxModules.moreheaders nginxModules.shibboleth ];
+  };
```

Comment:
I see it a little problematic to add new variant for every nginx module combination we have. Either we have an nginx packet, that includes the most modules we have or a version, which makes use of nginx [dynamic modules](https://www.nginx.com/blog/dynamic-modules-nginx-1-9-11/) architecture, where plugins become a shared library.